sway/commands/permit.c

```diff
@@ -54,6 +60,10 @@ static struct feature_policy *get_policy(const char *name) {
 		}
 		list_add(config->feature_policies, policy);
 	}
+	sway_assert(!strcmp(program, policy->program),
+			"!! DANGER !! symlink modified during policy allocation.");
```

Comment:
Time of check/Time of use. The symlink could be switched under us from the time we validate to the time we create the policy.